packages/nitro-protocol/gas-benchmarks/fixtures.ts

```diff
@@ -0,0 +1,15 @@
+import {Wallet} from '@ethersproject/wallet';
+
+import {Channel, getChannelId} from '../src';
+
+export const chainId = '0x1';
+export const channelNonce = 2;
+
+export const Alice = new Wallet(
+  '0x277fb9e0ad81dc836c60294e385b10dfcc0a9586eeb0b1d31da92e384a0d2efa'
+);
+export const Bob = new Wallet('0x277fb9e0ad81dc836c60294e385b10dfcc0a9586eeb0b1d31da92e384a0d2efa');
```

Comment:
Alice and Bob are constructed here from the same private key: 0x277...efa
- is this number significant? If not, why use it rather than 0x1?
- if I understand correctly, this will also mean that they have the same public key / address. Does that make sense?

Reply:
We should at least use distinct keys for Alice and Bob, I think. Will fix that. While it doesn't make much sense for an actual state channel that might get used, I don't think it makes any difference to the gas consumption to have the two the same (which is the main concern here). Still it's easy enough to have distinct ones, and less confusing. Also having two the same might confuse the gas accounting if we end up paying out money to Alice and Bob.

The number chosen for the private key is not significant: but I wanted to avoid magic values with many zeros in them, in case that were to lead to some unrealistic gas savings somewhere. I generated the private key using `ethers.Wallet.createRandom()`.

Reply:
Interestingly, changing Bob's private key did have a small effect on gas consumption 0161c5b. I think this is because of the follow-on change to the `channelId` used in the benchmarking test -- I'm not entirely sure why writing to the `holdings` mapping with a different key would cost a different amount of gas...